src/Analyzer.Cli.UnitTests/Analyzer.Cli.UnitTests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+	
+	<AssemblyName>Microsoft.Azure.Cli.Analyzer.Cli.UnitTests</AssemblyName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.8" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.8" />
+    <PackageReference Include="coverlet.collector" Version="3.1.2" />
+  </ItemGroup>
+
+	<ItemGroup>
```

Comment:
The indentation of this whole group also looks off?

Reply:
Shoot, Visual Studio mixed up tabs and spaces for some reason.  I'll fix that.